### ISSUE TITLE
build fails with aeson-2.0

### ISSUE THREAD

**Issue by juhp** [sternenseemann/spacecookie#51]
```
spacecookie                    > /tmp/stack-e91b7c5fd4ba17cb/spacecookie-1.0.0.0/server/Network/Spacecookie/Config.hs:28:28: error:
spacecookie                    >     • Couldn't match expected type ‘Key’ with actual type ‘Text’
spacecookie                    >     • In the second argument of ‘(.:?)’, namely ‘x’
spacecookie                    >       In the expression: v .:? x
spacecookie                    >       In an equation for ‘maybePath’: maybePath [x] v = v .:? x
spacecookie                    >    |
spacecookie                    > 28 | maybePath [x]    v = v .:? x
spacecookie                    >    |                            ^
spacecookie                    > 
spacecookie                    > /tmp/stack-e91b7c5fd4ba17cb/spacecookie-1.0.0.0/server/Network/Spacecookie/Config.hs:29:28: error:
spacecookie                    >     • Couldn't match expected type ‘Key’ with actual type ‘Text’
spacecookie                    >     • In the second argument of ‘(.:?)’, namely ‘x’
spacecookie                    >       In the first argument of ‘(>>=)’, namely ‘v .:? x’
spacecookie                    >       In the expression: v .:? x >>= fmap join . traverse (maybePath xs)
spacecookie                    >    |
spacecookie                    > 29 | maybePath (x:xs) v = v .:? x >>= fmap join . traverse (maybePath xs)
spacecookie                    >    |                            ^
```

**Comment by sternenseemann** [comment]
@juhp Fixed in 75275cf971197f1b1da4464b17e39129428e527b and uploaded `1.0.0.1` to hackage. Anything else I need to do?

**Comment by juhp** [comment]
Perfect!
